test/unittests/eof_validation_stack_test.cpp

```diff
@@ -1241,11 +1241,11 @@ TEST_F(eof_validation, jumpf_to_nonreturning)
 
     // Exactly required inputs on stack at JUMPF
     add_test_case(eof_bytecode(3 * OP_PUSH0 + jumpf(1), 3).code(OP_STOP, 3, 0x80, 3),
-        EOFValidationError::success);
+        EOFValidationError::unused_input);
```

Comment:
not necessarily urgent, but leaving a comment to not miss - we may maybe want to move such `unused_input` cases to their separate batch of tests, and rewrite these variants to actually have them use the inputs.

Or, maybe actually writing these tests now could be useful to get more insight into what the change entails.

Reply:
That's the plan. I only wanted to list all affected tests.